bucket/musicbee.json

```diff
@@ -2,8 +2,7 @@
     "version": "3.3.7491",
     "description": "The ultimate music player to easily manage, find, and play music, podcasts, web radio stations, etc",
     "homepage": "https://getmusicbee.com",
-    "license": "GPL-2.0-or-later",
-    "notes": "Currently it's not possible to make Plugins and Skins persistent",
```

Comment:
Why is it possible now?

Reply:
I think so. 
Scoop used to have a bug when persisting things that already existed (https://github.com/lukesampson/scoop/pull/2732) but this change was merged after the original manifest was created. (Nov 16, 2018)
Newer versions of musicbee come with the AppData and Library folder already, and scoop does not have a problem persisting them.

Reply:
How is the manifest addition date relevant to this line?? The persisting message was added in #2371, because scoop does not support merging the persisted folders, hence it cannot handle when developers add new native plugin or skins.

Reply:
I see. I thought the note was added shortly after the original manifest was created

Reply:
Revert everything related to persisting and keep only license change